inbc-program/README.md

```diff
@@ -478,7 +534,6 @@ inbc source application update
 ## SOURCE APPLICATION LIST
 ### Description
 Lists Application sources
-NOTE: Currently this only works on Ubuntu
 
 ### Usage
```

Comment:
(Not something changed in this PR but a related question)

Would the output of the list cmd include the gpg file name for the application source (if available)?
If not, then we should plan on a future PR to include that, as our remove cmd requires --gpgKeyName parameter for deleting the respective source.

Reply:
@pillaiabhishek - I'll let @gblewis1 answer that since he did the list implementation.

Reply:
@pillaiabhishek application list doesn't do so we would need a future PR

Reply:
@pillaiabhishek actually, I take that back; application list shows the contents of the .list file which has a pointer to the GPG file in it, so we would not need to change anything

Reply:
Great. Thanks for confirming.